Branch-SDK/src/io/branch/referral/ServerRequest.java

```diff
@@ -22,6 +22,8 @@
     /*True if there is an error in creating this request such as error with json parameters.*/
     public boolean constructError_ = false;
 
+    private final JSONObject getParamObject;
```

Comment:
Do we need to split out getParams from normal parameters on a request? If the request is type GET, can you just use the other JSONObject?